src/dug/core/crawler.py

```diff
@@ -85,13 +85,14 @@ def crawl(self):
     def annotate_elements(self):
 
         # Annotate elements/concepts and create new concepts based on the ontology identifiers returned
-        for element in self.elements:
+        logger.info(f"annotate {len(self.elements)} elements")
+        for n, element in enumerate(self.elements):
             # If element is actually a pre-loaded concept (e.g. TOPMed Tag), add that to list of concepts
             if isinstance(element, DugConcept):
                 self.concepts[element.id] = element
 
             # Annotate element with normalized ontology identifiers
-            self.annotate_element(element)
+            self.annotate_element(element, n+1, len(self.elements))
```

Comment:
It looks like the extra args are just used for logging, my thought is that it would be cleaner to add the logging here instead of farther down the stack

Reply:
Sure, will change it.  I forget that this is only single-threaded -- no concurrency.